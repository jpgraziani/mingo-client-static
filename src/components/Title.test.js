import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Title from './Title'

describe(`Title component`, () => {
  it('renders a Title by default', () => {
    const wrapper = shallow(<Title />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})