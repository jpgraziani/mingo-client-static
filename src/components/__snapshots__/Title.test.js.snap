// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component renders a Title by default 1`] = `
<div
  className="section-title"
>
  <h2 />
</div>
`;
